views/snippets/form_date.html

```diff
@@ -21,7 +21,7 @@ <h1 class="heading-small">What is your date of birth?</h1>
 
       <div class="form-group form-group-year">
         <label for="dob-year">Year</label>
-        <input class="form-control" id="dob-year" name="dob-year" type="number" pattern="[0-9]*" min="0" max="2014">
+        <input class="form-control" id="dob-year" name="dob-year" type="number" pattern="[0-9]*" min="1910" max="2014">
```

Comment:
Does this affect validation if a user enters their DOB using only the last two digits? eg `62`. Previously 62 would have been in the range but it's not now.